be/src/formats/parquet/group_reader.cpp

```diff
@@ -30,6 +30,15 @@ namespace starrocks::parquet {
 constexpr static const LogicalType kDictCodePrimitiveType = TYPE_INT;
 constexpr static const LogicalType kDictCodeFieldType = TYPE_INT;
 
+GroupReader::GroupReader(GroupReaderParam& param, int row_group_number, std::set<std::int64_t> need_skip_rowids)
+        : _need_skip_rowids(std::move(need_skip_rowids)), _param(param) {
+    _row_group_metadata =
+            std::make_shared<tparquet::RowGroup>(param.file_metadata->t_metadata().row_groups[row_group_number]);
+    for (int i = 0; i < row_group_number; ++i) {
+        _row_group_first_row += param.file_metadata->t_metadata().row_groups[i].num_rows;
```

Comment:
Since `FileReader` will iterate over groups, so it's better to keep count and set it by `FileReader`.